cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
I don't think we should start creating extra packages if this isn't used elsewhere. It ends up creating a dependency that may be under-designed.